src/js/config/Config.template.ts

```diff
@@ -11,6 +11,10 @@ export default {
         banner: {
           enabled: false
         },
+        intercom: {
+          enabled: true,
+          appId: "wn4z9z0y"
```

Comment:
💥 Can we please use something like "<appId from intercom>" here?

Reply:
I went with the code because we already have `analyticsKey: "39uhSEOoRHMw6cMR6st9tYXDbAL3JSaP"` in the same file and that would ease the friction testing intercom for every newcomer. It's a public key after all.